Formula/f3.rb

```diff
@@ -15,7 +13,13 @@ class F3 < Formula
     sha256 "96ee5681212139b960fdaca98839e2e5e23446f1b890b751c459b05bedabaf6a" => :mavericks
   end
 
+  head do
+    url "https://github.com/AltraMayor/f3.git"
+    depends_on "argp-standalone"
+  end
+
   def install
+    ENV.append "LDFLAGS", "-largp" if build.head?
```

Comment:
This is a bug that needs to be reported and fixed upstream.

Reply:
If someone wants to do it they can. I don't have the time or motivation to do so.

Reply:
OK, we'll probably just remove `head` in that case.